modules/@apostrophecms/schema/ui/apos/components/AposInputString.vue

```diff
@@ -121,17 +121,30 @@ export default {
           return 'required';
         }
       }
-      if (this.field.min) {
-        if (this.type === 'number' || this.type === 'date') {
+
+      const minMaxNumFields = [
```

Comment:
This makes it clearer and prevents us from needing to jam `range` in there some other way.